flink-kubernetes-webhook/src/test/java/org/apache/flink/kubernetes/operator/admission/AdmissionHandlerTest.java

```diff
@@ -52,7 +53,9 @@ public class AdmissionHandlerTest {
     private final AdmissionHandler admissionHandler =
             new AdmissionHandler(
                     new FlinkValidator(
-                            new DefaultValidator(new FlinkConfigManager(new Configuration()))));
+                            Set.of(
```

Comment:
This doesn't verify whether the custom validator implementation works well here.

Reply:
Fixed